shortlister/view.py

```diff
@@ -16,15 +16,17 @@ def view_role(self, role: Role):
         print(f"\nRole title: {role.job_title}")
         print(f"Role ID: {role.job_id}")
 
-    def view_applicant_details(self, applicant: Applicant):
+    def view_applicant_details(self, applicant: Applicant,criteria):
         """Prints details of applicant to console."""
         print(f"Details for {applicant.name}: ")
         print(f"CV Path: {applicant.cv}")
 
         if applicant.scores:
             print("Scores: ")
-            for criterion, score in applicant.scores.items():
-                print(f"{criterion.name:^20}: {score:^20}")
+            for order in criteria:
+                for criterion, score in applicant.scores.items():
+                    if criterion == order:
+                        print(f"{criterion.name:^20}: {score:^20}")
```

Comment:
You are looping through all the applicant's scores every time checking each criteria in the role. However, we can very easily check for the specific criteria looping over, and print it if it is present:

```suggestion
                if order in applicant.scores:
                        print(f"{order.name:^20}: {applicant.scores[order]:^20}")
```